.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -57,7 +57,7 @@ body:
       label: Version
       description: What version of our software are you running?
       options:
-        - 2.2.0 (Default)
+        - 2.2.1 (Default)
```

```suggestion
        - 2.3.0 (Default)
```

IMO, when doing semver, the patch number is for bug fixes, but new (compatible) features should increment the minor number.